.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: ruby
+rbenv:
+- 2.5.3
+addons:
+  postgresql: 9.6
+script:
+- bundle exec rails db:{create,migrate} RAILS_ENV=test
+- bundle exec rspec
+deploy:
+  provider: heroku
+  # api_key:
```

Comment:
Nice @LHJE! Did the CLI command for getting this API key work for you?

Reply:
It Did!  Thanks for helping with getting that implemented